packages/ui/src/common/components/statistics/StakeStat.tsx

```diff
@@ -0,0 +1,40 @@
+import BN from 'bn.js'
+import React from 'react'
+import styled from 'styled-components'
+
+import { MultiStatisticItem } from '@/common/components/statistics/MultiTokenValueStat'
+import { TextMedium } from '@/common/components/typography'
+import { Colors } from '@/common/constants'
+
+import { StatisticItemProps, StatisticItemSpacedContent } from './StatisticItem'
+import { TokenValueStat, TotalValue } from './TokenValueStat'
+
+export interface StakeStatProps extends StatisticItemProps {
+  value: number | BN
+  minStake: number
+}
+
+const title = 'Stake Height'
+
+export const StakeStat = ({ value, minStake }: StakeStatProps) => {
+  if (new BN(value).gte(new BN(minStake))) {
+    return <TokenValueStat value={value} title={title} />
+  }
+
+  return (
+    <MultiStatisticItem title={title}>
```

Comment:
I think that here you might use base stat item: [https://github.com/Joystream/pioneer/blob/8802903259051ccb8d2f9d0625aad9c543f790c5/packages/ui/src/common/components/statistics/StatisticItem.tsx#L18-L25](https://github.com/Joystream/pioneer/blob/8802903259051ccb8d2f9d0625aad9c543f790c5/packages/ui/src/common/components/statistics/StatisticItem.tsx#L18-L25)

There are props for the `(!)` tootlip (`helper...`).

Reply:
I used Multiple, because it already has 2 rows. Check new commit - I have introduced possibility to add custom icon for tooltip